collie/cross_validation.py

```diff
@@ -237,42 +239,53 @@ def _stratified_split(interactions: BaseInteractions,
                       processes: int,
                       seed: int,
                       force_split: bool) -> Tuple[Interactions, Interactions]:
-    users = interactions.mat.row
-    unique_users = set(users)
-
+    try:
+        if interactions.negative_sample_type == 'item':
+            users_or_items = interactions.mat.row
+        elif interactions.negative_sample_type == 'user':
+            users_or_items = interactions.mat.col
+    except AttributeError:
+        users_or_items = interactions.mat.row
```

Comment:
I think we can simplify this logic a bit more: 

```suggestion
    users_or_items = interactions.mat.row
    
    if getattr(interactions, 'negative_sample_type') == 'user':
        users_or_items = interactions.mat.col
```

Reply:
This raises an `AttributeError` I added a default `if getattr(interactions, 'negative_sample_type', 'item') == 'user':` let me know what you think in the next review.